infrastructure/demo/main/ecs.tf

```diff
@@ -46,22 +46,6 @@ resource "aws_service_discovery_service" "admin_app" {
   }
 }
 
-# vectordb service discovery service
-resource "aws_service_discovery_service" "vectordb" {
-  name = "vectordb"
-  dns_config {
-    namespace_id   = aws_service_discovery_private_dns_namespace.web.id
-    routing_policy = "MULTIVALUE"
-    dns_records {
-      ttl  = 10
-      type = "SRV"
-    }
-  }
-  health_check_custom_config {
-    failure_threshold = 1
-  }
-}
-
 
 # Nginx Service with EC2 Launch Type
 resource "aws_ecs_service" "nginx_service" {
```

Comment:
Are we planning on getting rid of the nginx service since we moving to Caddy?

Reply:
Yes! Either replace with Caddy or look into load balancer..